.github/workflows/release.yml

```diff
@@ -25,14 +27,14 @@ jobs:
 
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
-          repository: wingtk/gvsbuild
+          repository: ${{ github.repository_owner }}/gvsbuild
```

Comment:
This won't work unless we fork it here so what about an input var that defaults to `wingtk`?

Reply:
Indeed. This is because I had to make a few patches to get everything working.

Reply:
If you look at my fork, you'll see the heaviest patching was for making an updated environment that still works with python 3.7. However, there was also a patch on the newer branch to get around a bug in glib2

Reply:
Ok, so I just consolidated everything under a single "latest" release. It uses my fork for the 3.7 version and the original wingtk repository for the 3.9 and 3.10 builds. I've removed inputs entirely for the workflow.